tensorflow/compiler/tf2tensorrt/BUILD

```diff
@@ -99,6 +99,43 @@ cc_library(
     ] + if_tensorrt([":tensorrt_lib"]),
 )
 
+cc_library(
+    name = "testutils",
+    testonly = 1,
+    srcs = ["utils/trt_testutils.cc"],
+    hdrs = [
+        "utils/trt_testutils.h",
+    ],
+    copts = tf_copts(),
+    visibility = ["//tensorflow:internal"],
+    deps = [
+        ":trt_conversion",
+        "@com_google_absl//absl/strings",
+        "@com_google_googletest//:gtest",
+        "//tensorflow/core:protos_all_cc",
+        "//tensorflow/cc:cc_ops",
+        "//tensorflow/core/framework:tensor_testutil",
+    ] + if_tensorrt([":tensorrt_lib"]),
+)
+
+tf_cuda_cc_test(
+    name = "testutils_test",
+    size = "small",
+    srcs = ["utils/trt_testutils_test.cc"],
+    tags = [
+        "no_windows",
+        "nomac",
+    ],
+    deps = [
+        ":testutils",
+        "//tensorflow/core:test_main",
+        "//tensorflow/core:protos_all_cc",
+        "@com_google_protobuf//:protobuf",
```

Comment:
This line doesn't seem to work. Can you use this
https://github.com/tensorflow/tensorflow/blob/35571d62233eacb329f794aaa4ebe80d8720c3bf/tensorflow/core/BUILD#L321

"//tensorflow/core/platform:protobuf",